documentation/faq/10-what-can-i-make-with-sveltekit.md

```diff
@@ -0,0 +1,21 @@
+---
+title: What can I make with SvelteKit?
+---
+
+SvelteKit can be used to create most kinds of applications. Out of the box, SvelteKit supports many features including:
+
+- Dynamic page content with [loaders](/docs/load) and [API routes](/docs/routing#server).
+- SEO-friendly dynamic content with [server-side rendering (SSR)](/docs/glossary#ssr).
+- User-friendly progressively-enhanced interactive pages with SSR and [Form Actions](/docs/form-actions).
+
+SvelteKit can also be deployed to a wide spectrum of hosted architectures via [adapters](/docs/adapters). In cases where SSR is used or server-side logic is added without using prerendering, those functions will be adapted to the target back-end. Some examples include:
+
+- Self-hosted dynamic web applications with a [Node.js backend](/docs/adapter-node).
+- Serverless web applications with back-end loaders and APIs deployed as remote functions. See [zero-config deployments](/docs/adapter-auto) for popular deployment options.
+- [Static pre-rendered sites](/docs/adapter-static) such as a blog or multi-page site hosted on a CDN or static host. Statically-generated sites are shipped without a backend.
+- [Single-page Applications (SPAs)](/docs/single-page-apps) with client-side routing and rendering for API-driven dynamic content. SPAs are shipped without a backend and are not server-rendered. This option is commonly chosen when serving SvelteKit from a custom server written in C, Golang, Rust, etc.
+- A mix of the above; some routes can be static, and some routes can use back-end functions to fetch dynamic information. This can be configured with [page options](/docs/page-options) that includes the option to opt out of SSR.
+
+In order to support SSR, a Node.js back-end or Node.js-based serverless function is required. When targeting a custom or external back-end, and SSR is desired, it's possible to deploy SvelteKit alongside the external back-end using the ["Backend For Frontend (BFF)" pattern](https://samnewman.io/patterns/architectural/bff/).
```

Comment:
technically doesn't have to be Node.js. I don't think we should mention bff as architectural patterns seem a bit out-of=place on this page and there are other architectural patterns that could be used as well
```suggestion
In order to support SSR, a JS backend — such as Node.js or Deno-based server, serverless function, or edge function — is required.
```